jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
 
                     if len(concat_sql) > 1:
                         concat_sql = "CONCAT(" + ",".join(concat_sql) + ") AS " + _make_column_name(column_number)
                     else:
                         concat_sql = concat_sql[0] + " AS " + _make_column_name(column_number)
+                    
+                    query_tables = [t["nest"] for t in tables]
+                    if details.nested_path not in query_tables:
+                        p = details.nested_path
+                        alias = nest_to_alias[p]
+                        from_sql += (
+                            "\nLEFT JOIN\n" + quote_table(concat_field(base_table, p)) + " " + alias +
+                            " ON " + alias + "." + PARENT + " = " + previous.alias + "." + UID
+                        )
```

Comment:
This left join might work, but it needs more isolation from the rest of the query:  This left join will multiply with any other left join to create an exponential explosion in the number of records.  Be sure to left join with a table/view/resultset that has only **one row for each UID**.  This means joining with something like:

    (SELECT parent_id, group_concat(var) as unioned FROM nested_table GROUP BY parent_id)